app/models/ontology/class_methods_and_scopes.rb

```diff
@@ -0,0 +1,99 @@
+module Ontology::ClassMethodsAndScopes
+  extend ActiveSupport::Concern
+
+  included do
+    equal_scope 'repository_id'
+
+    scope :without_parent, where('ontologies.parent_id' => nil)
+
+    scope :basepath, ->(path) do
+      joins(:ontology_version).where('ontology_versions.basepath' => path)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.